stats/mpstat

```diff
@@ -90,11 +90,13 @@ rescue JSON::ParserError
   end
 end
 
+idle_pct = []
```

Comment:
thanks, one more thing to consult, what does "pct" mean?

Reply:
pct is for Percentage.

Reply:
got it, since i saw other stats uses style like node.all.idle%, probably this can be changed to max_utilization%, but not a must. I will merge the pr firstly.